web/src/routes/Event.jsx

```diff
@@ -18,9 +40,25 @@ export default function Event({ eventId }) {
 
   return (
     <div className="space-y-4">
-      <Heading>
-        {data.camera} {data.label} <span className="text-sm">{startime.toLocaleString()}</span>
-      </Heading>
+      <div className="flex">
+        <Heading className="flex-grow">
+          {data.camera} {data.label} <span className="text-sm">{startime.toLocaleString()}</span>
+        </Heading>
+        <Button className="self-start" color="red" aria-label="Delete Event" onClick={handleClickDelete}>
```

Comment:
Buttons should include text visibly whenever possible:

```js
<Button className="self-start" color="red" onClick={handleClickDelete}>
  <Delete className="w-6" /> Delete event
</Button>
```

Reply:
I originally took this code from the toggle buttons, so I was trying to be consistent with all the buttons without text. I can add text pretty easily.